lib/net_cash_flow/main.py

```diff
@@ -37,26 +36,27 @@ def get_net_cash_flow(
     # STEP 2: Loop through all transactions adding the transactions
     cash_in_amount = 0
```

Comment:
this should be moved to line 53.
initializing these vars, and their use, is only of concern to the area of code that manages the feature computation. so it should be moved there for clarity and structure.